lib/tasks/retention_instructions.rake

```diff
@@ -0,0 +1,54 @@
+# frozen_string_literal: true
+
+# $ bundle exec rake "retention_instructions:backfill[batch_size, limit]"
+require_relative '../../app/helpers/retention_instruction_helper'
+
+namespace :retention_instructions do
+
+  include RetentionInstructionHelper
+  desc 'Backfill retention instructions'
+
+  task :backfill, [:batch_size, :limit] => :environment do |_, args|
+    args.with_defaults(batch_size: 1000, limit: nil)
+    batch_size = args[:batch_size].to_i
+    limit = args[:limit]&.to_i
+
+    puts "Backfilling retention instructions with batch size: #{batch_size}..."
+
+    ActiveRecord::Base.transaction do
+      saved_count = 0
+      labwares = Labware.where(retention_instruction: nil)
+      labwares = labwares.limit(limit) unless limit.nil?
```

Comment:
@yoldas A `limit` parameter just to test the script in UAT. This is by default `nil` which prompts to fetch all the records.

Reply:
You could add `.order(:id)` to get the labwares in a predictable order to help testing and to quickly know where you would start next time. It is a good idea and it looks good otherwise.